README.md

```diff
@@ -1,86 +1,113 @@
 
-[![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors-)
+This library offers a simple contract to use the biometry across Android, iOS and Windows (UWP & WinUI).
 
-<!-- ALL-CONTRIBUTORS-BADGE:END -->
+## Getting Started
 
-This library offers a simple contract to use the biometry across Android, iOS and Windows (UWP & WinUI).
+1. Install the latest stable version of `BiometryService` in your platform heads and `BiometryService.Abstractions` in your presentation layer if you are using MVVM pattern, and if not just install both in your platform heads.
+   
+1. Instanciate BiometryService.
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+   Every platform has its unique biometryService implementation.
 
-## Features
+   ### Windows
 
-The Biometry Service Interface `IBiometryService` is made of the following methods:
+   - Simply create a new BiometryService.
 
-- GetGapabilites
-- ScanBiometry
-- Encryt
-- Decrypt
-- Remove
+   ``` cs
+    _biometryService = new BiometryService();
+   ```
 
-As of now, this is the list of features available per platform.
+   ### Android
 
-| Feature          | iOS     | Android | UWP     | WinUI   |
-| ---------------- | ------- | ------- | ------- | ------- |
-| GetCapability    | &check; | &check; | &check; | &check; |
-| ValidateIdentity | &check; | &check; | &cross; | &cross; |
-| Encrypt          | &check; | &check; | &cross; | &cross; |
-| Decrypt          | &check; | &check; | &cross; | &cross; |
+   - Face authentication is only available when using `.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricWeak)` in the `BiometricPrompt.PromptInfo.Builder` instantiation that is required for the service. Please note that if you are using `.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricStrong)` in the `BiometricPrompt.PromptInfo.Builder` Facial authentification is exclusively accessible on phones equipped with Class 3 Biometric capabilities. (Pixel 4 and 8 for now).
 
-## Getting Started
+   - Please note that Encrypt/Decrypt methods are only available when using `.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricStrong)` in the `BiometricPrompt.PromptInfo.Builder` instantiation that is required for the service.
 
-Install the latest stable version of `BiometryService` in your platform heads and `BiometryService.Abstractions` in your presentation layer if you are using MVVM pattern, and if not just install both in your platform heads.
+   - Please also note that the title and subtitle are used for `Fingerprint` and `Face` biometry.
 
-A small sample Uno application is available as a playground with some basic command to test the service methods.
-They also provide some basic initialization but no dependency injection and more complex code.
+   - Here is an example of instantiation of the service for Android.
```

Comment:
In the getting started section, you should be more direct because you're giving directions.
Instructions should be phrased more like "Do this" or "Set that" instead of "Here's an example" or "Note that you can do it this way".
That way, the person following the steps should not have any questions. He just follows the instructions and he'll have a minimal working solution.

The Features section can offer more options and is more appropriate for things like "If you want X, you can do Y".